test/test_blah.py

```diff
@@ -0,0 +1,46 @@
+import torch
```

Comment:
Is there an existing test that could be used as an example instead of these samples?

Reply:
https://github.com/pytorch/pytorch/blob/c74c0c571880df886474be297c556562e95c00e0/test/test_autograd.py#L4301 is parameterized over fast_mode in [True, False]

https://github.com/pytorch/pytorch/blob/c74c0c571880df886474be297c556562e95c00e0/test/test_autograd.py#L3915-L3923 is also parametrized in the same way (a lot of the other gradcheck tests are this way too)

Reply:
To add a data point... I like having the samples in test_blah as test cases because if something starts failing then it is clearer the failure is due to the parameterize functionality and not e.g. a bug in PyTorch operator code

Reply:
We can add tests to test_testing.py (there are some examples) if we want to preserve testing the decorator without a specific test

Reply:
Yes, I definitely agree on adding tests to `test_testing.py` for these decorators. To @zou3519's point, being able to distinguish parametrization failures vs. real test failures is crucial.

Just dumped them in test_blah.py for the sake of demonstration, but they should definitely be moved to a proper permanent location :)